sdks/python/apache_beam/io/range_trackers.py

```diff
@@ -285,6 +309,14 @@ def fraction_consumed(self):
                        ' consumed due to positions being opaque strings'
                        ' that are interpreted by the service')
 
+  def split_points(self):
+    with self._lock:
+      splits_points_consumed = (
+          0 if self._split_points_seen <= 1 else (self._split_points_seen - 1))
+
+      return (splits_points_consumed,
+              iobase.RangeTracker.SPLIT_POINTS_UNKNOWN)
```

Comment:
I think, shuffle position is an opaque set of bytes (as far as Beam SDK is considered) so don't think think there is a way to determine if a given key the last one before the stop position (I could be missing something here).

BTW this RangeTracker is Dataflow runner specific and should be removed from Beam. Will handle that in a separate CL.